shell/browser/api/atom_api_url_loader.cc

```diff
@@ -29,6 +29,28 @@
 #include "shell/common/gin_helper/object_template_builder.h"
 #include "shell/common/node_includes.h"
 
+namespace gin {
+
+template <>
+struct Converter<mojo::InlinedStructPtr<network::mojom::HttpRawHeaderPair>> {
```

Comment:
```suggestion
struct Converter<network::mojom::HttpRawHeaderPairPtr> {
```

Reply:
The type of `response_head.raw_request_response_info` is `std::vector< mojo::InlinedStructPtr<network::mojom::HttpRawHeaderPair>>`, the `mojo::InlinedStructPtr` can not be omitted here otherwise compilation error would happen.

Reply:
I'm confused,

```
../out/Testing/gen/services/network/public/mojom/http_raw_headers.mojom-forward.h
29:using HttpRawHeaderPairPtr = mojo::InlinedStructPtr<HttpRawHeaderPair>;
```

these two types are defined to be identical. how could one cause a compile error?

Reply:
Sorry I did not notice the `Ptr` in class name.